_examples/complete/main.tf

```diff
@@ -190,14 +188,14 @@ module "addons" {
   aws_node_termination_handler = true
   aws_efs_csi_driver           = true
   aws_ebs_csi_driver           = true
-  karpenter                    = false
-  calico_tigera                = false
+  karpenter                    = true
+  calico_tigera                = true
+  new_relic                    = false # Update the licence key before using this add-on from ./config/override-new-relic.yaml
```

Comment:
can you please mention this comment in addon's Readme.md and remove it from here.